addons/website/static/src/builder/plugins/options/bento_banner_option.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="html_builder.BentoBannerOption">
+    <BuilderRow label.translate="Roundness">
+        <BuilderRange action="'setClassRange'" actionParam="['rounded-0','rounded-1','rounded-2','rounded-3','rounded-4','rounded-5']" max="5"/>
+    </BuilderRow>
```

Comment:
Seems weird: we have a "global roundness" task, which will be about using classes for roundness etc. Should not we wait for that before doing this and meanwhile, just use the standard option "Rounded Corners" that comes with default columns?

Reply:
https://github.com/odoo/odoo/pull/208793#discussion_r2154735181 -> alright, let's leave it as you want and we'll review with the roundness task later.